CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### 6.2.0 (2022-10-20)
 
 #### Bugs
+* Fix #4535: The shell command string will now have single quotes sanitized
```

Comment:
Please move this to the 6.3 section above.  

Unfortunately we don't yet have an automated way of generating these entries as part of the release process.